packages/web-components/tests/e2e-storybook/cypress/integration/pricing-table/_tests.js

```diff
@@ -46,6 +51,33 @@ export const createTests = path => [
       });
     });
   },
+  () => {
+    it('should render user-entered content next to the icon feature, when it is enabled', () => {
+      const iconText = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc dui magna, finibus id tortor sed, aliquet bibendum augue.'
+
+      cy.visit(`${path}&knob-icon-text=${iconText}`)
+      .get('.cds--pricing-table-cell-content').should('be.visible').then(($content) => {
+        cy.get($content)
+          .find('svg')
+          .should('be.visible')
+          .then(($svg) => {
+            const svgRect = $svg[0].getBoundingClientRect();
+
+            cy.get($content)
+              .find('.cds--structured-list-cell-icon-text')
+              .should('be.visible')
+              .then(($span) => {
+                const spanRect = $span[0].getBoundingClientRect();
+                
+                // Checking if the elements are rendered side by side by comparing if they have the same vertical position, allowing a tolerance of 15 px of difference.
+                expect(spanRect.top).to.be.closeTo(svgRect.top, 15);
+                // Same logic as above but now to check if all content is vertically aligned.
+                expect(spanRect.left).to.be.closeTo(svgRect.left, 1);
```

Comment:
Is this asserting that the icon and the text should be vertically aligned? Because I don't think that's the goal of the second case we want to test for. Rather, we want to make sure that all the text in the cells of a given column are vertically aligned, specifically when only some of them have also are configured to show the icon.

In other words, given the following table we want to make sure all the text in the first column is vertically aligned even though the second cell doesn't use an icon.

| Column 1 | Column 2 | Column 3 |
|-|-|-|
| ✅ Lorem ipsum dolor sit amet | Foo | Bar |
| Consectetur adipiscing elit | Foo | Bar |
| ✅ Cras fermentum luctus magna | Foo | Bar |

Reply:
After discussing on a Slack huddle earlier today, we're going to remove this test case. The v1 component does not support this vertical alignment feature at this time.